controller/mariadb_controller_test.go

```diff
@@ -130,33 +130,33 @@ var _ = Describe("MariaDB controller", func() {
 			}, testTimeout, testInterval).Should(BeTrue())
 
 			By("Expecting to create a exporter Deployment eventually")
-			Eventually(func() bool {
+			Eventually(func(g Gomega) bool {
 				var deploy appsv1.Deployment
 				if err := k8sClient.Get(testCtx, testMariaDb.MetricsKey(), &deploy); err != nil {
 					return false
 				}
 				expectedImage := os.Getenv("RELATED_IMAGE_EXPOTER")
-				Expect(expectedImage).ToNot((BeEmpty()))
+				g.Expect(expectedImage).ToNot(BeEmpty())
 				By("Expecting Deployment to have exporter image")
-				Expect(deploy.Spec.Template.Spec.Containers).To(ContainElement(MatchFields(IgnoreExtras,
+				g.Expect(deploy.Spec.Template.Spec.Containers).To(ContainElement(MatchFields(IgnoreExtras,
 					Fields{
 						"Image": Equal(expectedImage),
 					})))
 				return deploymentReady(&deploy)
-			})
+			}).Should(BeTrue())
```

Comment:
```suggestion
			}, testTimeout, testInterval).Should(BeTrue())
```
These `Eventually` block is wrong, missing timeouts. Again, not sure why it did work before.

Reply:
It's not totally wrong because there are default polling interval and timeout, but I fixed to match this specific test.